src/main/java/seedu/pill/command/HelpCommand.java

```diff
@@ -1,33 +1,92 @@
 package seedu.pill.command;
 
+import seedu.pill.exceptions.PillException;
 import seedu.pill.util.ItemList;
 import seedu.pill.util.Storage;
 
 /**
  * Represents a command that displays help information about available commands.
  */
 public class HelpCommand extends Command {
+    private String commandName;
+
+    public HelpCommand() {
+        this.commandName = null;
+    }
+
+    public HelpCommand(String commandName) {
+        this.commandName = commandName;
+    }
 
     /**
      * Executes the help command by displaying information about available commands.
-     *
-     * @param itemList The item list (not used in this command).
+     * @param itemList The item list to be manipulated by the command.
+     * @param storage
+     * @throws PillException
      */
     @Override
-    public void execute(ItemList itemList, Storage storage) {
+    public void execute(ItemList itemList, Storage storage) throws PillException {
```

Comment:
I like that you abstracted out methods in execute such that it is easily understood and more readable